TLM/TLM/Manager/Impl/LaneConnection/LaneConnectionManager.cs

```diff
@@ -24,7 +24,7 @@ public class LaneConnectionManager
                                                              | VehicleInfo.VehicleType.Trolleybus;
 
         public LaneConnectionSubManager Sub = // TODO #354 divide into Road/Track
-            new LaneConnectionSubManager(LaneEndTransitionGroup.All);
+            new LaneConnectionSubManager(LaneEndTransitionGroup.Vehicle);
```

Comment:
@krzychu124 when cherry picking, this will cause conflict. just ignore the change to non-existing file.